examples/systemcalls/systemcalls.c

```diff
@@ -9,13 +28,9 @@
 */
 bool do_system(const char *cmd)
 {
-
-/*
- * TODO  add your code here
- *  Call the system() function with the command set in the cmd
- *   and return a boolean true if the system() call completed with success
- *   or false() if it returned a failure
-*/
+	//check if system call succeeded
+	if(system(cmd) == -1)
```

Comment:
@PeterBraganza is `-1` the right check here or are there other cases where you need to return false per the comments on the function?